templates/flavors/k3s/secret.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: linode-${CLUSTER_NAME}-crs-0
+stringData:
+  linode-token-region.yaml: |-
+    kind: Secret
+    apiVersion: v1
+    metadata:
+      name: linode-token-region
+      namespace: kube-system
+    stringData:
+      apiToken: ${LINODE_TOKEN}
+      region: ${LINODE_REGION}
+  cilium.yaml: |-
+    apiVersion: helm.cattle.io/v1
+    kind: HelmChart
+    metadata:
+       namespace: kube-system
+       name: cilium
+    spec:
+       targetNamespace: kube-system
+       version: v1.15.0
```

Comment:
can we have this version be a variable? https://github.com/linode/cluster-api-provider-linode/blob/main/templates/addons/cilium/cilium.yaml#L12